contrib/docs-examples/snippets/react-native/GoogleCredential.js

```diff
@@ -0,0 +1,98 @@
+function example() {} function untested({View, Text, Button, React, GoogleSignin, Stitch, GoogleCredential, GoogleSigninButton}) { // too much boilerplate to be able to test this
+  // Example component that uses the react-native-google-signin module to get
+  // the server auth code for Stitch to use.
+  //
+  // NOTE: The react-native-google-signin native module must be installed correctly
+  // and your Google project must be configured.
+  //
+  // For more detail, see https://stackoverflow.com/questions/55145071/#55173164
+  //
+  // Above:
+  // import {Stitch, GoogleCredential} from 'mongodb-stitch-react-native-sdk'
+  // import {GoogleSignin, GoogleSigninButton} from 'react-native-google-signin'
+  class GoogleLogin extends React.Component {
+    // ...
```

Comment:
nit: maybe we should be more explicit here?

e.g. `... other component methods`